README.md

```diff
@@ -272,52 +197,42 @@ and one indicating a stability "dev". For example:
 }
 ```
 
-## The "lock file” and the "package hash"
+## The "lock file" and the "package hash".
 
-Everytime the assets are processed for a package, a lock file named **`.composer_compiled_assets`**
-is saved in the package's root.
+Each time the assets for a package are processed, a lock file named **`.composer_compiled_assets`** is saved in the root directory of the package.
 
-That file should be git-ignored if the compiled assets are git-ignored.
+This file should be git-ignored if the compiled assets are git-ignored.
```

Comment:
I think "That" is more appropriate in this case. It refers to something that has mentioned before in a separate sentence, and using "This" does not make the reference clear, IMO.

Maybe, it would be better being more explicit like:

> `.composer_compiled_assets`  should be git-ignored if the compiled assets are git-ignored.

Reply:
I'm not a native speaker, but `that` sounds wrong to me. I changed it anyway.